### PR TITLE
HTML: Merge file structure

### PR DESCRIPTION
**Pull request by garett09**

### CODE REVIEW

File: css/sample.css [hunk 1]
```diff
@@ -0,0 +1,57 @@
+/**
```
Comment (garett09) [inline css/sample.css:1]: This is ok since this is only a sample

File: css/sample.css [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+@import url('https://codedotspectra.github.io/themes/core/hljs.css');
+@import url('https://codedotspectra.github.io/themes/core/settings-icons.css');
+@import url('https://discordstyles.github.io/RadialStatus/base.css');
+@import url('https://codedotspectra.github.io/themes/core/badges.css');
+/* Fixes */
+@import url('https://xcruxiex.github.io/BBDThemes/Fixes/NocturnalFixes.css');
+/*#################################################################################################*/
+:root {
+  --backgroundDark: rgb(24, 28, 37);
+  --backgroundDarker: rgb(20, 23, 31);
```
Comment (garett09) [inline css/sample.css:36]: What are these colors?

File: index.html [hunk 3]
```diff
@@ -0,0 +1,12 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>AFK Apartelle</title>
+</head>
+<body>
```
Comment (garett09) [inline index.html:9]: test comment

File: css/sample.css [hunk 4]
```diff
@@ -0,0 +1,57 @@
+/**
+ * @name Nocturnal
```
Comment (eduardonacion) [inline css/sample.css:2]: WRONG

Comment (garett09) [reply css/sample.css:2]: i can see

**Review by eduardonacion — APPROVED** [review]
GOOD

**Review by Stateria — APPROVED** [review]

**Pull request merged**